### PR TITLE
SPINEDEM-3403 - Removed stateful behaviour of Karate sandbox

### PR DESCRIPTION
**Pull request by agelledi**
Removed stateful behaviour of Karate sandbox and updated addAndRemove.feature related tests.

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.

### CODE REVIEW

File: karate-tests/src/test/java/mocks/stubs/patientResponses/patient_9733162051.json
Comment (PeterGallagherBJSS) [inline karate-tests/src/test/java/mocks/stubs/patientResponses/patient_9733162051.json:1]: Is this used somewhere (or meant to be)? There's the scenario in addAndRemove.feature but not marked as sandbox I think

**Review by PeterGallagherBJSS — COMMENTED** [review]
Is this branch for covering all stateful behaviour of the sandbox? 
If so there's also some code for this in post-patient.js and a few other places that needs removing.

File: karate-tests/src/test/java/mocks/stubs/patientResponses/patient_9733162051.json
Comment (agelledi) [reply karate-tests/src/test/java/mocks/stubs/patientResponses/patient_9733162051.json:1]: Sorry missed to mark scenario as sandbox

**Comment by agelledi** [comment]
> Is this branch for covering all stateful behaviour of the sandbox? If so there's also some code for this in post-patient.js and a few other places that needs removing.

Yes it's suppose to cover all stateful behaviour of the sandbox, I will take a look at post-patient.js

**Review by PeterGallagherBJSS — APPROVED** [review]

**Pull request merged**